wemake_python_styleguide/violations/best_practices.py

```diff
@@ -2849,3 +2851,33 @@ class ImportObjectCollisionViolation(ASTViolation):
 
     error_template = 'Found import object collision: {0}'
     code = 474
+
+
+@final
+class RedundantTernaryViolation(ASTViolation):
+    """
+    Forbid the redundant ternary operat which returns sanme value.
+
+    Reasoning:
+        There is no need to use a ternary operator to return the same value
+        for both the true and false cases.
+    Solution:
+        Instead of using a ternary operator, use the value directly.
```

Comment:
```suggestion
    Reasoning:
        If the ternary expression can be simplified,
        it should be simplified.

    Solution:
        Instead of using a ternary operator, use the value directly.
```